packages/redux-dynostore-react-redux/test/dynamic.test.js

```diff
@@ -87,4 +87,32 @@ describe('dynamic tests', () => {
     expect(wrapper.html()).toBe('<p>Hello World</p>')
     expect(callback).toHaveBeenCalledTimes(1)
   })
+
+  test('should create instance of dynamic component with enhancer and instance enhancer', () => {
+    const callback = jest.fn()
+
+    const fakeStore = {}
+
+    const testEnhancer1 = identifier => store => {
```

Comment:
Not necessary but you could avoid duplicating the enhancer code by making this

```js
const testEnhancer = instance => identifier => store => {
  expect(identifier).toBe('testId')
  expect(store).toBe(fakeStore)
  callback(instance)
}
```
and then calling them like
```js
const DynamicComponent = dynamic('baseId', testEnhancer(1))(TestComponent).createInstance('testId', testEnhancer(2))
```

Reply:
Sure